swiftsimio/reader.py

```diff
@@ -85,15 +102,60 @@ def get_unit_dictionary(self):
 class SWIFTMetadata(object):
     """
     Loads all metadata (apart from Units, those are handled by SWIFTUnits)
-    into dictionaries. This also does some extra parsing on some well-used
-    metadata.
+    into dictionaries. 
+    
+    This also does some extra parsing on some well-used metadata.
+
+    Attributes
+    ----------
+    filename: str
+        name of file to read from
+    units: SWIFTUnits
+        the units being used
+    header: dict
+        stores metadata about snapshot
+
+    Methods
+    -------
+    get_metadata(self):
+        Loads the metadata 
+    get_named_column_metadata(self):
+        Loads custom metadata from named columns
+    postprocess_header(self):
+        Postprocesses local variables in header 
+    present_particle_types(self):
+        Get particle types present in snapshot
+    present_particle_names(self):
+        Get particle names present in snapshot
+    code_info(self):
```

Comment:
These info values are not in fact methods - they are again properties and so should be documented under `Attributes`.